og.module

```diff
@@ -169,6 +170,47 @@ function og_entity_access(EntityInterface $entity, $operation, AccountInterface
   return AccessResult::forbidden();
 }
 
+/**
+ * Implements hook_ENTITY_TYPE_access().
+ */
+function og_og_membership_type_access(OgMembershipTypeInterface $entity, $operation, AccountInterface $account) {
+  // Do not allow deleting the default membership type.
+  if ($operation === 'delete' && $entity->id() === OgMembershipInterface::TYPE_DEFAULT) {
+    return AccessResult::forbidden();
+  }
+
+  // If the user has permission to administer all groups, allow access.
+  if ($account->hasPermission('administer group')) {
+    return AccessResult::allowed();
+  }
+
+  return AccessResult::forbidden();
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_access().
+ */
+function og_og_membership_access(OgMembershipInterface $entity, $operation, AccountInterface $account) {
+  $group = $entity->getGroup();
+
+  // If there's a group owner, don't let them leave.
+  if (
+    isset($group_fields['uid'])
```

Comment:
`$group_fields` is not defined anywhere

Reply:
@jludwig, I suggest something like this:
if (
```suggestion
    $group instanceof EntityOwnerInterface
    && $group->getOwnerId() === $entity->getOwnerId()
```